Assets/Scripts/Game/UI/DialogueSystem.cs

```diff
@@ -88,8 +88,21 @@ private void OnDialogueOptionClicked(int index)
         {
             DialogueGraphTransition[] transitions = GetTransitionsFor(currentGraph, currentNode);
             DialogueGraphTransition transition = transitions[index];
-            if (transition.to < 0) dialogueUI.SetActive(false);
-            else StartCoroutine(ShowDialogue(transition.to));
+            if (transition.to < 0)
+            {
+                dialogueUI.SetActive(false);
+                return;
```

Comment:
I don't think this return statement is necessary; we could probably remove it.

Reply:
I mostly added it as a future-proofing so we don't run any more transition code if we don't have a valid "to" node. But I agree, it mostly just adds confusion because it doesn't do anything right now. I'll remove it. 👍